Add answers for multiple choice questions only

Cellaryllis opened the pull request.

Cellaryllis committed:
Add answers for multiple choice

Cellaryllis committed:
Fix wrong correct answer

poddus commented:
thanks for the interest! just fyi this project didn't end up getting very far before I lost interest :sweat_smile: and I'm not really intending on working on it any further.
it looks like you took `answers_with_correct.txt` and removed the text questions to make a new file, is that correct?

Cellaryllis commented:
> thanks for the interest! just fyi this project didn't end up getting very far before I lost interest 😅 and I'm not really intending on working on it any further. it looks like you took `answers_with_correct.txt` and removed the text questions to make a new file, is that correct?

Hey, it's okay if you don't want to work on it further, I just wanted to update it since I took your preparsed data and used it for a simple app I made for myself (I will make it open source once it's done). And I figured if I had use for your (pre)parsed data, maybe others do to so might as well PR over a complete list of correct answers.

And yes, correct, I took that list and removed all non-multiple-choice questions and then continued marking correct answers.

poddus commented:
sounds good to me!
